src/runtime/process.rs

```diff
@@ -549,3 +555,37 @@ impl Default for LinuxSchedulerFlag {
         LinuxSchedulerFlag::SchedResetOnFork
     }
 }
+
+#[derive(Builder, Clone, Debug, Deserialize, Getters, Setters, Eq, PartialEq, Serialize)]
+#[builder(
+    default,
+    pattern = "owned",
+    setter(into, strip_option),
+    build_fn(error = "OciSpecError")
+)]
+#[getset(get = "pub", set = "pub")]
+/// ExecCPUAffinity specifies CPU affinity used to execute the process.
+/// This setting is not applicable to the container's init process.
+pub struct ExecCPUAffinity {
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    /// cpu_affinity_initial is a list of CPUs a runtime parent process to be run on
+    /// initially, before the transition to container's cgroup.
+    /// This is a a comma-separated list, with dashes to represent ranges.
+    /// For example, `0-3,7` represents CPUs 0,1,2,3, and 7.
+    cpu_affinity_initial: Option<String>,
+
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    /// cpu_affinity_final is a list of CPUs the process will be run on after the transition
+    /// to container's cgroup. The format is the same as for `initial`. If omitted or empty,
+    /// the container's default CPU affinity, as defined by cpu.cpus property, is used.
+    cpu_affinity_final: Option<String>,
+}
+
+impl Default for ExecCPUAffinity {
+    fn default() -> Self {
+        Self {
+            cpu_affinity_initial: None,
+            cpu_affinity_final: None,
+        }
+    }
+}
```

Comment:
I think we can just derive `Default`.

Reply:
@saschagrunert thanks for reviewing this PR!
Without this implementation, I got this error.

```bash
cargo check
   Compiling serde_json v1.0.118
    Checking oci-spec v0.6.5 (/home/ubuntu/workspace/oci-spec-rs)
error[E0277]: the trait bound `ExecCPUAffinity: Default` is not satisfied
   --> src/runtime/process.rs:559:10
    |
559 | #[derive(Builder, Clone, Debug, Deserialize, Getters, Setters, Eq, PartialEq, Serialize)]
    |          ^^^^^^^ the trait `Default` is not implemented for `ExecCPUAffinity`
    |
    = note: this error originates in the derive macro `Builder` (in Nightly builds, run with -Z macro-backtrace for more info)
help: consider annotating `ExecCPUAffinity` with `#[derive(Default)]`
    |
569 + #[derive(Default)]
570 | pub struct ExecCPUAffinity {
    |

For more information about this error, try `rustc --explain E0277`.
error: could not compile `oci-spec` (lib) due to 1 previous error
```

Reply:
What about ` #[derive(…, Default)]`?

Reply:
@saschagrunert sorry, I see what you mean now. It worked!